### PR TITLE
feat(ext/crypto): verify ECDSA signatures

### PR DESCRIPTION
**Pull request by littledivy**
Towards #11690 

~~Blocked on #11738~~

### CODE REVIEW

File: ext/crypto/00_crypto.js [hunk 1]
```diff
@@ -717,6 +718,25 @@
...
           }, data);
         }
+        case "ECDSA": {
+          // 1.
+          if (key[_type] !== "public") {
+            throw new DOMException(
+              "Key type not supported",
+              "InvalidAccessError",
+            );
+          }
```
Comment (bnoordhuis) [inline ext/crypto/00_crypto.js:728]: Can you exercise this code path with a test?

(Mea culpa, I may have forgotten to point that out in your other pull requests.)

**Review by bnoordhuis — APPROVED** [review]

File: ext/crypto/00_crypto.js (hunk 1 above)
Comment (lucacasonato) [reply ext/crypto/00_crypto.js:728]: I am not 100% sure, but this is likely covered by WPT tests.

Comment (bnoordhuis) [reply ext/crypto/00_crypto.js:728]: The same thought occurred to me the moment I hit "Submit review". :-)

Comment (littledivy) [reply ext/crypto/00_crypto.js:728]: Yup WPT covers this...https://github.com/denoland/wpt/blob/146f12e8df2cac6b1e60152145124a81dad60d38/WebCryptoAPI/sign_verify/ecdsa.js#L238 (using a "secret" key to sign ECDSA)
Adding WPT is pending

Comment (littledivy) [reply ext/crypto/00_crypto.js:1147]: uhhh so WPT depends on importKey. I've added a unit test for now

**Review by ry — APPROVED** [review]
LGTM- thanks @littledivy !

**Pull request merged**